<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班票务系统 - 用户界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>640</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="searchTab">
     <attribute name="title">
      <string>航班查询</string>
     </attribute>
     <widget class="QGroupBox" name="searchGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>950</width>
        <height>100</height>
       </rect>
      </property>
      <property name="title">
       <string>查询条件</string>
      </property>
      <widget class="QLabel" name="fromLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>60</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>出发地:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="fromCombo">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>120</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="toLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>60</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>目的地:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="toCombo">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>120</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="dateLabel">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>60</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>出发日期:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>30</y>
         <width>120</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>30</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="transferButton">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>30</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>转机查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="flightTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>950</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ticketTab">
     <attribute name="title">
      <string>我的机票</string>
     </attribute>
     <widget class="QTableWidget" name="ticketTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>950</width>
        <height>590</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reservationTab">
     <attribute name="title">
      <string>预约抢票</string>
     </attribute>
     <widget class="QTableWidget" name="reservationTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>950</width>
        <height>590</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionProfile"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProfile">
   <property name="text">
    <string>个人信息</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 